src/app/api/boatInfo/[id]/route.ts

```diff
@@ -0,0 +1,31 @@
+import { Client } from "@notionhq/client";
+
+const notionDatabaseId: string | undefined = process.env.NOTION_DATABASE_ID;
+const notionSecret: string | undefined = process.env.NOTION_SECRET;
+
+// Initialize a new client with the secret
+const notion = new Client({ auth: notionSecret });
+
+export async function GET(request: Request, { params }: { params: { id: string } }) {
```

Comment:
Instead of exporting a function `GET`

Lets export a singletion class Notion.get().bookingInfo(bookingId)

The `request` parameter is not used here, not sure why we needed